.github/workflows/build.yml

```diff
@@ -13,38 +13,32 @@ jobs:
     strategy:
       matrix:
         config:
-         - {grb_version: 7.1.0, conda_grb_package_hash: h27087fc, conda_extension: tar.bz2}
-         - {grb_version: 7.3.0, conda_grb_package_hash: h27087fc, conda_extension: tar.bz2}
-         - {grb_version: 7.4.1, conda_grb_package_hash: hcb278e6, conda_extension: conda}
+         - {grb_version: 9.3.1}
     steps:
     - name: Checkout
       uses: actions/checkout@v2.0.0
     - name: Install tools for build
       run: |
         sudo apt install -y lcov
-    - name: Get GraphBLAS binaries
+    - name: Build GraphBLAS
       run: |
-        mkdir graphblas-binaries
-        cd graphblas-binaries
-        wget --quiet https://anaconda.org/conda-forge/graphblas/${{ matrix.config.grb_version }}/download/linux-64/graphblas-${{ matrix.config.grb_version }}-${{ matrix.config.conda_grb_package_hash }}_0.${{ matrix.config.conda_extension }}
-        if [ ${{ matrix.config.conda_extension }} == "tar.bz2" ]; then
-          tar xf graphblas-${{ matrix.config.grb_version }}-${{ matrix.config.conda_grb_package_hash }}_0.${{ matrix.config.conda_extension }}
-        else
-          unzip graphblas-${{ matrix.config.grb_version }}-${{ matrix.config.conda_grb_package_hash }}_0.${{ matrix.config.conda_extension }}
-          tar xf pkg-graphblas-${{ matrix.config.grb_version }}-${{ matrix.config.conda_grb_package_hash }}_0.tar.zst
-        fi
+        git clone https://github.com/DrTimothyAldenDavis/GraphBLAS.git
+        cd GraphBLAS
+        git checkout tags/v${{ matrix.config.grb_version }}
+        make compact
+        sudo make install
         cd ..
     - name: Build project
       run: |
-        export GRAPHBLAS_INCLUDE_DIR=`pwd`/graphblas-binaries/include
-        export GRAPHBLAS_LIBRARY=`pwd`/graphblas-binaries/lib/libgraphblas.so
+        export GRAPHBLAS_INCLUDE_DIR=`pwd`/GraphBLAS/include/suitesparse
+        export GRAPHBLAS_LIBRARY=`pwd`/GraphBLAS/lib/libgraphblas.so
         cd build
         cmake .. -DCOVERAGE=1 -DGRAPHBLAS_INCLUDE_DIR=${GRAPHBLAS_INCLUDE_DIR} -DGRAPHBLAS_LIBRARY=${GRAPHBLAS_LIBRARY}
         JOBS=2 make
         make test_coverage
     - name: Deploy
       uses: JamesIves/github-pages-deploy-action@4.1.1
-      if: matrix.config.grb_version == '7.4.1' && github.event_name == 'push' && github.ref == 'refs/heads/stable'
+      if: matrix.config.grb_version == '9.1.0' && github.event_name == 'push' && github.ref == 'refs/heads/stable'
```

Comment:
This should read '9.3.1', or the test "matrix.config.grb_version == '9.1.0'" should be removed.  Not sure which.